### PR TITLE
Correct a number of trivial typos.

### PR DESCRIPTION
**Pull request by lamby**

### CODE REVIEW

**Comment by poettering** [comment]
lgtm

**Comment by lamby** [comment]
I hope I made the Hofstadterian distinction between `renable` and "re-enable". :)

**Comment by keszybz** [comment]
I don't think that this distinction matters here for spelling with or without a dash. If anything it's better to use `"reenable"` (with quotes), when the word itself is being mentioned.

**Comment by lamby** [comment]
> when the word itself is being mentioned.

You mean when the *command* itself is being mentioned. Use-mention distinction and all that.

(Can you point to an example in the diff where this is violated? Otherwise I'm not sure what I'm being asked to do here.)

**Comment by keszybz** [comment]
I'm asking for both the command and the action to be spelled the same: reenable.

**Comment by lamby** [comment]
Done :)

File: src/systemctl/systemctl.c [hunk 1]
```diff
@@ -7087,7 +7087,7 @@ static void systemctl_help(void) {
                "  list-unit-files [PATTERN...]        List installed unit files\n"
                "  enable [UNIT...|PATH...]            Enable one or more unit files\n"
                "  disable UNIT...                     Disable one or more unit files\n"
-               "  reenable UNIT...                    Reenable one or more unit files\n"
+               "  reenable UNIT...                    Re-enable one or more unit files\n"
```
Comment (keszybz) [inline src/systemctl/systemctl.c]: Here too.

Comment (lamby) [reply src/systemctl/systemctl.c]: Whoops, sorry. :)

Comment (lamby) [reply src/systemctl/systemctl.c]: (done)

**Comment by keszybz** [comment]
Thanks.

**Pull request merged**